tests/test_e2e_40_sdntrace.py

```diff
@@ -424,8 +416,4 @@ def test_030_run_sdntrace_for_stored_flows(cls):
         response = requests.put(api_url, json=payload)
         assert response.status_code == 200, response.text
         data = response.json()
-        assert len(data) == 2
-        assert "00:00:00:00:00:00:00:01" in data
-        assert len(data["00:00:00:00:00:00:00:01"]) == 2
-        assert "00:00:00:00:00:00:00:0a" in data
-        assert len(data["00:00:00:00:00:00:00:0a"]) == 1
+        assert len(data["result"]) == 3
```

Comment:
@gretelliz looks good, this ensures at least a basic regression test. 

I'll encourage you to further confirm that the result responses are in order as well, and checking some of the fields of the responses.